src/components/Swap/Skeleton.tsx

```diff
@@ -37,44 +36,25 @@ const InputColumn = styled(Column)`
   margin-bottom: 0.25em;
   padding: 0.75em;
   padding-bottom: 3.25em;
-  padding-top: 1.5em;
+  padding-top: 3.25em;
 `
 
-const OutputColumn = styled(Column)`
-  background-color: ${({ theme }) => theme.module};
-  border-radius: ${({ theme }) => theme.borderRadius - 0.25}em;
-  display: flex;
-  padding-top: 0.5em;
-`
-
-const OutputInnerTopColumn = styled(Column)`
-  border-bottom: 1px solid ${({ theme }) => theme.container};
-  padding-bottom: 3.25em;
-  padding-left: 0.75em;
-  padding-right: 0.75em;
-  padding-top: 1.5em;
-  width: 100%;
-`
-
-const OutputInnerBottomColumn = styled(Column)`
-  padding-bottom: 1em;
-  padding-left: 0.75em;
-  padding-right: 0.75em;
-  padding-top: 0.75em;
-  width: 100%;
+const OutputColumn = styled(InputColumn)`
+  padding-bottom: 3em;
+  padding-top: 3.5em;
 `
 
 const ButtonColumn = styled(Column)`
   padding-bottom: 0em;
-  padding-top: 0.75em;
+  padding-top: 0.55em;
```

Comment:
should this be 0.5em?

Reply:
This needs to be 0.55em or it will be off by 1px.

Reply:
It's hard to see unless you disable the brandedFooter.